frontend/src/router/index.tsx

```diff
@@ -1,11 +1,11 @@
-import { Dashboard, Login, Signup, UserProfile } from 'pages';
+import { Main, Login, Signup, UserProfile } from 'pages';
 
 export const routes = [
   {
-    name: 'Dashboard',
+    name: 'Main',
     path: '/',
     requireAuth: true,
-    component: <Dashboard />,
+    component: <Main />,
```

Comment:
It is bad naming for "Main" component. I know that application will be large then one page, that it should be name like Home page or Landing or Auth page example